doc/reference/reference_lua/fiber.rst

```diff
@@ -63,6 +63,18 @@ Below is a list of all ``fiber`` functions and members.
     | :ref:`fiber.testcancel()             | Check if the current fiber has  |
     | <fiber-testcancel>`                  | been cancelled                  |
     +--------------------------------------+---------------------------------+
+    | :ref:`fiber.set_max_slice()          | Set the default maximum slice   |
+    | <fiber-set_max_slice>`               | for all fibers                  |
+    +--------------------------------------+---------------------------------+
+    | :ref:`fiber.set_slice()              | Set a slice for the current     |
+    | <fiber-set_slice>`                   | fiber                           |
+    +--------------------------------------+---------------------------------+
+    | :ref:`fiber.extend_slice()           | Extend a slice for the          |
+    | <fiber-extend_slice>`                | current fiber                   |
```

Comment:
`set_slice` and `extend_slice` are not about current fiber. They are about current fiber EXECUTION. Current fiber execution ends after a `fiber.yield()`. Example:
```lua
local function fib_f()
    operation_1()
    fiber.set_slice(5)
    operation_2()
    fiber.yield()
    operation_3()
    fiber.extend_slice(4)
    operation_4()
end
```
Imagine that we have default err slice = 1 second. Then, we will have limitation in 1 second for 1st operation. After that, we set err slice 5 seconds for a current fiber execution, that means that we have limitation in 5 seconds for operation 1 and operation 2.
Then `fiber.yield()` happens, and slices are set to defaults. So, we have 1 sec limitation for operation 3, then slice of current execution is extended to 5 sec (1 sec by default + 4 sec by extent) and we have 5 sec limitation for operation 3 + operation 4.

Reply:
thanks to the detailed explanation, now I got it